jerry-main/main-unix-snapshot.c

```diff
@@ -92,7 +92,7 @@ static size_t
 read_file (uint8_t *input_pos_p, /**< next position in the input buffer */
            const char *file_name) /**< file name */
 {
-  FILE *file = fopen (file_name, "r");
+  FILE *file = fopen (file_name, "rb");
```

Comment:
The same comment goes here as for main-unix.c and main-unix-test.c: the caller of `read_file` should decide whether its a text or binary mode open.

Reply:
The reason for always reading a binary file is to avoid any line ending conversion on Windows, even when we are reading text files.